nodes/src/nodes/anomaly_detector/IInstance.py

```
@@ -0,0 +1,161 @@
+
+        if self._metric == 'sentiment_score':
+            return _try_parse_sentiment(text)
+
+        extractor = _METRIC_EXTRACTORS.get(self._metric)
+        if extractor is None:
+            warning(f'anomaly_detector: unknown metric {self._metric!r}')
+            return None
+
+        try:
+            return extractor(text)
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)